tests/unit/CurrencySymbolUtilsTest.js

```diff
@@ -0,0 +1,34 @@
+// Taken from GetCurrencyList API.
```

Comment:
I think this comment should be on JSON file.

Reply:
I think we can not add comments in JSON file